app/src/main/java/com/example/keniphhomemaintenance/dwellings/DwellingsScreenView.kt

```diff
@@ -15,12 +23,12 @@ import com.example.keniphhomemaintenance.ui.cookbook.KeniphFloatingActionButton
 @Composable
 fun DwellingsScreenView(
     navController: NavController,
-    dwellingsViewModel: DwellingsViewModel = viewModel()
+    dwellingsViewModel: DwellingsViewModel = viewModel(
+        factory = DwellingsViewModelFactory(LocalContext.current)
```

Comment:
factory was needed to pass context to the viewModel. Not needed once dependency injection is implemented.